frontend/src/pages/account/NoTeamsUser.vue

```diff
@@ -1,41 +1,31 @@
 <template>
-    <div class="forge-block flex flex-col justify-center">
-        <div class="sm:max-w-xl mx-auto w-full space-y-2">
-            <div class="max-w-xs mx-auto w-full mb-4">
-                <FlowForgeLogo/>
-                <h2 class="mt-2 text-center text-3xl font-bold text-gray-900">
-                    <span>FLOW</span><span class="font-light">FORGE</span>
-                </h2>
+    <main>
+        <form class="px-4 sm:px-6 lg:px-8 mt-4 space-y-4 text-center">
+            <p class="text-gray-700 text-lg">You need to be in a team to create projects.</p>
+            <p v-if="invitationCount === 0 && !user.admin && !settings['team:create']" class="text-gray-700 text-lg mt-10 ">Ask an Admin or Team Owner to add you to a team.</p>
```

Comment:
I'm not a Vue expert, but is it possible to create helpers that capture logic like:

`invitationCount === 0 && !user.admin && !settings['team:create']`? That makes logic as such more reusable

Reply:
In theory, yes, although not sure it's appropriate here? What is a short-hand way of describing this particular set of conditions? Also, not sure we have use for this set of conditions elsewhere?

Reply:
I don't know if Vue has a best practice around this.

There is part of me that thinks trying to avoid too much logic in the template is worthwhile - by which I mean having long compound statements if the `v-if` property as is the case here.

If all of the 'logic' is held in computed properties it means there's a standard place to look in each component for those bits of logic - rather than have them scattered around the template itself (which could lead to things being overlooked when we have to modify the logic).

Reply:
>  I do fear that we're going to end up with a lot of computed variables which may make readability more difficult.

I think that's the interesting point. My instinct is having lots of well named computed variables would make the template more readable - as you don't have to parse the full logic to figure out what it is doing. For example `canCreateTeam` is understandable at a glance, rather than trying parse all the conditions in the expression - and if you need to check the logic, you have one place to look up how it is defined.

But lets not hold this PR up for this ongoing discussion.